data-models/common/acceleration/schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/location/acceleration/schema.json",
+    "title": "Acceleration",
+    "description": "Acceleration measurement.",
+    "type": "object",
+    "$defs": {
+        "accX": {
+            "measurementType": "location/acceleration/accX/mG",
+            "title": "Acceleration X",
+            "description": "Acceleration on the x-axis.",
+            "type": "number",
+            "unit": "mG"
+        },
+        "accY": {
+            "measurementType": "location/acceleration/accY/mG",
+            "title": "Acceleration Y",
+            "description": "Acceleration on the y-axis.",
+            "type": "number",
+            "unit": "mG"
+        },
+        "accZ": {
+            "measurementType": "location/acceleration/accZ/mG",
+            "title": "Acceleration Z",
+            "description": "Acceleration on the z-axis.",
+            "type": "number",
+            "unit": "mG"
+        }
+    }
```

Comment:
I guess we do not have a `required` here, as we could also only have 1-dimensional movements, right?

Reply:
yes, it is not yet clear for me how we will be using this exactly (a common measurementType _acceleration_ that is then used three times for x, y, z vs having a measurement type for an acceleration in the x-direction)

Reply:
Hmmm I guess we could keep the three fields as otherwise we or the users would need to always adjust the `measurementType`, `title` and `description` -- which are quite a lot of fields and arguably more error-prone then just providing it here and have a bit of duplication?